test/test_docker_utils.py

```diff
@@ -28,3 +32,34 @@ def test_add_registry_and_tag():
         assert utils.add_registry_and_tag(base_image) == 'default-registry/confluentinc/example:default-tag'
         assert utils.add_registry_and_tag(base_image, scope="UPSTREAM") == 'upstream-registry/confluentinc/example:upstream-tag'
         assert utils.add_registry_and_tag(base_image, scope="TEST") == 'test-registry/confluentinc/example:test-tag'
+
+
+@pytest.fixture(scope="module")
+def ecr_login():
+    # in AWS env, do a login.
+    if os.getenv('AWS_DEFAULT_REGION') is not None:
+        utils.ecr_login()
+
+
+@pytest.fixture(scope="module")
+def official_image(ecr_login):
+    utils.pull_image(OFFICIAL_IMAGE)
+
+
+@pytest.mark.integration
+def test_image_exists(official_image):
+    assert utils.image_exists(OFFICIAL_IMAGE)
+
+
+@pytest.mark.integration
+def test_path_exists_in_image(official_image):
+    assert utils.path_exists_in_image(OFFICIAL_IMAGE, "/usr/local/bin/dub")
+    assert utils.path_exists_in_image(OFFICIAL_IMAGE, "/usr/local/bin/cub")
+
+
+@pytest.mark.integration
+def test_run_docker_command(official_image):
```

Comment:
official_image doesn't seem used, is that intended?

Reply:
this uses pytest fixture to pull image which is necessary to start container and run commands in it.